CONTRIBUTING.md

```diff
@@ -161,10 +161,9 @@ This address can be opened in your browser to preview the tutorial website.
 
 General discussion of the tutorial happens in
 ESMValTool [User Engagement Team][user-engagement].
-You can reach us by email.
-Please see the [contact information][contact-info].
+You can reach us by [email][email].
 
-[email]: mailto:TODO_FIX_ME
+[email]: mailto:esmvaltool@listserv.dfn.de
```

Comment:
I think you may have to subscribe to the mailinglist before you're allowed to send it an email, maybe add a link to https://docs.esmvaltool.org/en/latest/community/contact.html#user-mailing-list or explain this?

Reply:
Also in all the other places where the email address is mentioned.

Reply:
I fixed it in all places.